README.md

```diff
@@ -172,155 +214,11 @@ Based on the size and strucutre of the dataset provided + these parameter settin
-```
-
-## Track Dataset Metrics
-The metrics associated with this dataset will be printed in the terminal. These metrics give some insight into how the data was packed into sequences, and information about the training dataset.
-| Metric Name      | Definition | How to Interpret? |
-| --------- | --------- | --------- |
-| Articles | The number of lines in the input dataset. | How many text documents in the input dataset. |
-| Dataset Tokens | Number of tokens in the output hdf5 dataset. | How many tokens are in the training dataset. But this includes both prompt tokens and padding tokens, so this metric does not necessarily show how many tokens will learned by the model. |
-| Prompt Tokens | Number of prompt tokens in the output hdf5 dataset. | <- |
-| Completion Tokens | Number of completion tokens in the output hdf5 dataset. | <- |
-| Padding Tokens | Number of padding tokens in the output hdf5 dataset. | <- |
-| Average Completion Length | Number of completion tokens divided by number of input articles. | How long the average completion is the dataset. |
-| Average Prompt Length | Number of prompt tokens divided by number of input articles. |  How long the average prompt is the dataset. |
-| Data Utilization | Percent of non-padding tokens in output HDF5 dataset divided by number of tokens in input dataset. | This metric reveals how much of the input data makes it to the output dataset. If this percent is much less than 100%, that means a lot of the input data will not be trained on. Refer to the "Dropped From Packing" or "Dropped From All Prompt" metrics to see why this is happening. |
-| Dropped From Packing  | Number of tokens dropped during packing, divided by number of tokens in input dataset. | The percent of tokens are dropped because they do not fit into the sequence length, and the `input_packing_config` does not allow them to be overflowed.|
-| Dropped From All Prompt | Number of tokens dropped because all the tokens in a sequence are prompt tokens, divided by the number of tokens in input dataset. | Sequences that are all prompts or padding (no completion tokens) are dropped. This is because the model will not learn anything from these sequences and the loss will be 0, which may cause errors. |
-| Sequence Utilization | Average number of non-padding tokens in a sequence divided by sequence length. | The percent of the tokens in each sequence are actually used for training. This number imrpoved be changed by using different `input_packing_config`. The packing styles from highest sequence utilization to lowest are: `full`, `greedy::truncate_left` (or truncate_right), `greedy::drop`, `single::truncate_left` (or truncate_right), `single::drop`.|
-| Seq Completion Utilization | Average number of completions tokens in a sequence divided by sequence length. | The percent of the tokens in a sequence are learned.|
-
-## Validating training parameters with dataset metadata
```

Comment:
I moved dataset validation with pydantic and sha256 checksum to advanced usage since I figure if the customer is building an integration with the data prep toolkit, they will fall into the bucket of an advanced user. For most users however, I imagine they just want to upload a dataset for fine-tuning so won't need to go through this section.